cmd/kubectl-moco/cmd/credential.go

```diff
@@ -21,6 +21,10 @@ var credentialCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) error {
 		return fetchCredential(cmd.Context(), args[0])
 	},
+	ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+		ctx := context.Background()
```

Comment:
How about using `cmd.Context()` in `mysqlClusterCandidates()`?
If you do so, setting up a ValidArgsFunction can be more simple.
```
ValidArgsFunction: mysqlClusterCandidates
```

In kubectl-moco, `cmd.Context()` is the same as `context.Background()`.
https://github.com/spf13/cobra/blob/v1.6.1/command.go#L983

Reply:
Thanks!:
https://github.com/cybozu-go/moco/pull/470/commits/0b6f626402a65d5567139fc301ee6348ee1bdea3